changelog.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - \[Demo\] Update application name and description in about page ([#419](https://github.com/Orange-OpenSource/ods-android/issues/419))
+- \[Demo\] Update application name and customization in progress Indicator ([#362](https://github.com/Orange-OpenSource/ods-android/issues/362))
+- \[Lib\] Update `OdsCircularProgressIndicator` and `OdsLinearProgressIndicator` components ([#362](https://github.com/Orange-OpenSource/ods-android/issues/362))
```

Comment:
I think you should be more precise in the comment for the lib. The users need to understand what has changed exactly. Something like: Add a `label` parameter to `OdsCircularProgressIndicator` in order to display a text below the indicator.
You can add another [Lib] line to explain changes for the `OdsLinearProgressIndicator`

Reply:
These change lines should be in the `### Changed` section. If there is no `### Changed` section you can add it.